Assets/Scripts/Control&Input/ItemSelectManager.cs

```diff
@@ -24,7 +29,13 @@ public void StartTrackingMenus()
 
     private void Finish()
     {
-        // TODO Fade to loading screen
+        StartCoroutine(LoadAndChangeScene());
+    }
+
+    private IEnumerator LoadAndChangeScene(){
+
+        loadingScreen.SetActive(true);
+        yield return new WaitForSeconds(delayDuration);
```

Comment:
Perhaps the delay should be placed after the loading (which is async) has started?

Reply:
FYI: 
- The method called here in matchcontroller doesn't load the the sene async (Even though it now should)

- If it did and you want the delay after such a load, one would have to write a hole block of code to stop Unity from automatically not send us to the next scene immediately.
From the Unity docs:
`
//Begin to load the [Scene](https://docs.unity3d.com/ScriptReference/SceneManagement.Scene.html) you specify
        [AsyncOperation](https://docs.unity3d.com/ScriptReference/AsyncOperation.html) asyncOperation = [SceneManager.LoadSceneAsync](https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.LoadSceneAsync.html)("[Scene](https://docs.unity3d.com/ScriptReference/SceneManagement.Scene.html)3");
        //Don't let the Scene activate until you allow it to
        asyncOperation.allowSceneActivation = false;
`

We can use the delay to toggle that property () to guarantee atleast 3 seconds have been spent in loading no matter the load times

Reply:
We'll leave this for later then